app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt

```diff
@@ -194,4 +197,13 @@ class StoryActivityTest {
 
     override fun getApplicationInjector(): ApplicationInjector = component
   }
+
+  // Mocking ImageLoader here, because Robolectric doesn't work with RenderScript, which is created
+  // in GlideImageLoader.
+  @Module
+  class TestModule {
+    @Provides
+    @Singleton
+    fun provideMockLoader() = Mockito.mock(ImageLoader::class.java)!!
```

Comment:
Prefer: ``provideMockImageLoader`` (to ensure the provider name includes the type).

Also nit: prefer statically importing ``mock``--this is a bit more conventional.

Ditto in StoryFragmentTest.

Reply:
Done and done.